nvbench/launch.cuh

```diff
@@ -43,9 +43,9 @@ struct launch
 
   // move-only
   launch(const launch &)            = delete;
-  launch(launch &&)                 = default;
+  launch(launch &&)                 = delete;
```

Comment:
Ah, the const ref member is an issue...

The "move-only" comment needs to be updated.

Reply:
Kept the move constructor and comment